packages/web/src/pages/profile-page/components/desktop/ProfilePage.tsx

```diff
@@ -391,7 +401,14 @@ const ProfilePage = ({
         )}
       </div>,
       <div key={ProfilePageTabs.PLAYLISTS} className={styles.cards}>
-        {playlists.length === 0 && !isOwner ? (
+        {collectionStatus !== Status.SUCCESS &&
```

Comment:
not `=== Status.LOADING`?

Reply:
idle & loading

Reply:
i did this direction just b /c it conforms to surrounding code

Reply:
gotcha. I probably prefer `IDLE || LOADING` but tbh all of this gets fixed anyway come tanstack/query 🤷